Connecting to database specified by database.yml


Started GET "/assets/rails.png" for 127.0.0.1 at 2014-12-08 11:49:47 +0200
Connecting to database specified by database.yml
Served asset /rails.png - 200 OK (4ms)


Started GET "/bagisci" for 127.0.0.1 at 2014-12-08 11:49:55 +0200

ActionController::RoutingError (No route matches [GET] "/bagisci"):
  /usr/lib/ruby/vendor_ruby/action_dispatch/middleware/debug_exceptions.rb:21:in `call'
  /usr/lib/ruby/vendor_ruby/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  /usr/lib/ruby/vendor_ruby/rails/rack/logger.rb:32:in `call_app'
  /usr/lib/ruby/vendor_ruby/rails/rack/logger.rb:16:in `block in call'
  /usr/lib/ruby/vendor_ruby/active_support/tagged_logging.rb:22:in `tagged'
  /usr/lib/ruby/vendor_ruby/rails/rack/logger.rb:16:in `call'
  /usr/lib/ruby/vendor_ruby/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.5.2) lib/rack/methodoverride.rb:21:in `call'
  rack (1.5.2) lib/rack/runtime.rb:17:in `call'
  /usr/lib/ruby/vendor_ruby/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.5.2) lib/rack/lock.rb:17:in `call'
  /usr/lib/ruby/vendor_ruby/action_dispatch/middleware/static.rb:63:in `call'
  /usr/lib/ruby/vendor_ruby/rails/engine.rb:484:in `call'
  /usr/lib/ruby/vendor_ruby/rails/application.rb:231:in `call'
  rack (1.5.2) lib/rack/content_length.rb:14:in `call'
  /usr/lib/ruby/vendor_ruby/rails/rack/log_tailer.rb:17:in `call'
  rack (1.5.2) lib/rack/handler/webrick.rb:60:in `service'
  /usr/lib/ruby/1.9.1/webrick/httpserver.rb:138:in `service'
  /usr/lib/ruby/1.9.1/webrick/httpserver.rb:94:in `run'
  /usr/lib/ruby/1.9.1/webrick/server.rb:191:in `block in start_thread'


  Rendered /usr/lib/ruby/vendor_ruby/action_dispatch/middleware/templates/rescues/routing_error.erb within rescues/layout (3.4ms)
